tox.ini

```diff
@@ -1,9 +1,8 @@
 [tox]
 skipsdist = True
 envlist =
-    py{27,34,35}-django{18,19,110,111}
```

Comment:
The support for `py36` has been added in #140, can you please also remove the changes to this file?

Reply:
Hi @romgar - thanks for taking a look! 

Since enums weren't introduced until Python 3.4, I'm not sure how to make enums compatible with this library, which also supports Python 2.7. I believe it's possible to use the `enum34` library which backports enums, and do a conditional install of `enum34` for any version of Python < 3.4. I've looked through the `tox` documentation and haven't found how to configure that. Any suggestions appreciated!

Reply:
Hey @cvlong ! Sorry for the (very!) late answer...

I recommend you to read this section: https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-settings
In this case, you would need to add a line in the `deps` section of the `[testenv]` to explicitly load the `enum34` library for python2:

```
deps =
    -rtests-requirements.txt
    django111: Django>=1.11,<1.11.99
    django20: Django>=2.0,<2.0.99
    django21: Django>=2.1,<2.1.99
    django22: Django>=2.2,<2.2.99
    py27: enum34
```